docs/tutorial/02-draw-a-snake.md

```diff
@@ -151,20 +165,20 @@ alias Scenic.Graph
 @tile_size 32
 ```
 
-Running the app, you won't see any change yet. We have to push our graph to the view port when we initialize the scene. We will also return the initial state we just set up:
+Running the app, you won't see any change yet. We have to push our graph to the viewport when we initialize the scene. We will also return the initial state we just set up:
 
 ```elixir
 def init(_arg, opts) do
   viewport = opts[:viewport]
 
   {:ok, %ViewPort.Status{size: {vp_width, vp_height}}} = ViewPort.info(viewport)
 
-  num_tiles_width = trunc(vp_width / @tile_size)
-  num_tiles_height = trunc(vp_height / @tile_size)
+  number_of_columns = div(vp_width, @tile_size)
```

Comment:
💚 great suggestion! Slipped my eyes we're still using `tiles` and a truncated variable name here before 🙈